sparkles/core.py

```diff
@@ -902,6 +903,22 @@ def check_acq_p2(self):
         elif P2 < P2_lim + 1:
             self.add_message('warning', f'P2: {P2:.2f} less than {P2_lim + 1} for {obs_type}')
 
+    def check_include_exclude(self):
+        """Check for included or excluded guide or acq stars (info)
+
+        """
+        call_args = self.call_args
+        for typ in ('acq', 'guide', 'fid'):
```

Comment:
I suppose since fids aren't supported yet, that the API hasn't been defined.  But this seems reasonable.  Method docstring could mention fids I suppose.

Reply:
Yes, when fids are included they will follow the same API.  I wanted to put this in now so we don't need to remember later.  I'll update the docstring though.